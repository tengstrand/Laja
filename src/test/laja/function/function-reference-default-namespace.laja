#namespace x

#write "target/test-output/function-reference-default-namespace.out"
  #loadClass("java.lang.Number") into number
  number.name = {number.name}
#end
