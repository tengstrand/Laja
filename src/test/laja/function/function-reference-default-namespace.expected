  number.name = java.lang.Number
